src/homework3/Task3.java

```diff
@@ -0,0 +1,50 @@
+package homework3;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class Task3 {
+    public static void main(String[] args) throws IOException {
+        HashMap<Integer, String> readFromFile = readFileToHashMap("D:\\LITS\\Homeworks\\TestFile.txt");
```

Comment:
```suggestion
        Map<Integer, String> readFromFile = readFileToHashMap("D:\\LITS\\Homeworks\\TestFile.txt");
```
Also try not to use full path to get the resource, because it will work only on your local PC, try to use relative paths next time

Reply:
Also try not to use full path to get the resource, because it will work only on your local PC, try to use relative paths next time.

Reply:
fixed in Homework3 fixes